src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -1519,6 +1520,14 @@ public String toVariableNameString(EnchantmentOffer eo) {
 					.description("The inventory close reason in an <a href='/events.html#inventory_close'>inventory close event</a>.")
 					.requiredPlugins("Paper")
 					.since("INSERT VERSION"));
+
+    if (Skript.isRunningMinecraft(1, 20))
+			Classes.registerClass(new EnumClassInfo<>(Side.class, "signside", "sign sides")
+					.user("sign ?sides?")
+					.name("Sign Sides")
+					.description("Represents a side of a sign.")
+					.requiredPlugins("Spigot 1.20+")
```

Comment:
We're missing a closing bracket from `Classes.registerClass( ... )` so this can't build.